src/ResourceManager/Sql/Commands.Sql/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Fixed issue where using Update-AzureRmSqlDatabaseVulnerabilityAssessmentSettings with storage account name parameter set
```

Comment:
This message isn't clear - it seems like you either didn't finish the sentence or meant "Fixed issue with using Update-AzureRmSqlDatabaseVulnerabilityAssessmentSettings with storage account name parameter set".  Please fix.